app/views/potential_match_settings/_potential_match_settings_form.html.erb

```diff
@@ -25,7 +25,7 @@
       <dd><%= match_settings_form.select(:num_required_prompts, PotentialMatchSettings::REQUIRED_MATCH_OPTIONS.select {|k,v| k != "0"}, :selected => potential_match_settings.num_required_prompts) %></dd>
 
       <% TagSet::TAG_TYPES.map {|t| t.pluralize }.each do |type| %>
-        <dt class="<%= type.downcase %>"><%= match_settings_form.label("num_required_#{type}", "#{type.capitalize}:") %></dt>
+        <dt class="<%= type.downcase %>"><%= match_settings_form.label("num_required_#{type}", "#{type.humanize.titleize}:") %></dt>
```

Comment:
I think the `class` attributes here and on line 40 should use `tag_type_class(type)`.